index.html

```diff
@@ -25,5 +25,7 @@ <h1>CitiPix</h1>
 					<input type="submit" value="Update" id="submit-btn">
 				</form>
 			</div>
+		<script src="http://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="js/script.js"></script>		
```

Comment:
Watch your indentation. These two `<script>` elements should be indented to the same level.